app/views/home/_project-category.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="mb-8">
+  <div class="flex mb-4 px-4 sm:px-0">
```

Comment:
There's kind of a large gap between the category title and the cards. Removing the `mb-4` class would get rid of the bottom margin. Not that important, maybe just try and see how it looks.